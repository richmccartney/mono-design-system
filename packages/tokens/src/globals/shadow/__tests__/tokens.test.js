const shadowTokens = require('../tokens.json');

describe('shadow tokens', () => {
  test(': matches snapshot', () => {
    expect(shadowTokens).toMatchSnapshot();
  });
});
