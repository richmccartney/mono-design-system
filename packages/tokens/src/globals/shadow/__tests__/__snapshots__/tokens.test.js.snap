// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shadow tokens : matches snapshot 1`] = `
Object {
  "shadow": Object {
    "level": Object {
      "100": Object {
        "value": "0px 2px 4px rgba(39,51,51,.24)",
      },
      "200": Object {
        "value": "0px 4px 8px rgba(39,51,51,.24)",
      },
      "300": Object {
        "value": "0px 8px 16px rgba(39,51,51,.24)",
      },
      "400": Object {
        "value": "0px 16px 32px rgba(39,51,51,.24)",
      },
    },
  },
}
`;
